lib/libzfs/libzfs_sendrecv.c

```diff
@@ -939,10 +961,18 @@ send_progress_thread(void *arg)
 			    tm->tm_hour, tm->tm_min, tm->tm_sec,
 			    (u_longlong_t)bytes, zhp->zfs_name);
 		} else {
-			zfs_nicebytes(bytes, buf, sizeof (buf));
-			(void) fprintf(stderr, "%02d:%02d:%02d   %5s   %s\n",
-			    tm->tm_hour, tm->tm_min, tm->tm_sec,
-			    buf, zhp->zfs_name);
+			if (pa->pa_parsable) {
+				(void) fprintf(stderr,
+				    "%02d:%02d:%02d\t%llu\t%s\n",
+				    tm->tm_hour, tm->tm_min, tm->tm_sec,
+				    (u_longlong_t)bytes, zhp->zfs_name);
```

Comment:
I don't think this can be executed, because the `pa_parsable` case is handled a few lines above.